spec/factories/organizations.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :organization do
+    title { "Computer Academy" }
```

Comment:
Since we'll want a unique title, we can use a sequence in the factory definition:

```suggestion
    sequence(:title) { |n| "Computer Academy #{n}" }
```